### ISSUE TITLE
Placement of the package origin/anchor

### ISSUE THREAD

**Issue by fruchti** [fruchti/horizon-library-convention#3]
Currently, there is no rule where the origin of a package should be. For reference: KiCAD has the following rules:

- For SMT, the anchor should be in the center, if the datasheet doesn't explicitly specify one (http://kicad-pcb.org/libraries/klc/F6.2/).
- For THT, it should be at pin 1 (http://kicad-pcb.org/libraries/klc/F7.2/).

Personally, I'd use the exact same rule for SMT components.

I'm not sure if the THT rule is the best one here, though. While it apparently has it's reasons in the pick and place business, it is pretty inconvenient in my opinion.

- Rotating parts is awkward.
- Electromagnetical parts like pushbuttons, potentiometers come with a very logical choice for the footprint origin. Also, a well-placed origin would be very convenient for edge-aligned connectors.

The only discussion of the KiCAD people about this issue I found is [this one](https://github.com/KiCad/kicad-library/issues/357). It doesn't go into much detail and I don't know much about pick and place, so it would be best if someone knew how much freedom in choosing the origin we actually have.

**Comment by atoav** [comment]
I agree with fruchtis sentiment. A centered rotational pivot is what I expect in all cases, except for parts like vertical potentiometers or jacks, where you want your origin to represent the center of the potentiometer axis or the plugged cable.  

I cant really say what the deal is with pin 2 origins for pick and place tho

**Comment by carrotIndustries** [comment]
I'm on your page in that regard. All packages should have their origin roughly in the center, not at pin 1.

**Comment by fruchti** [comment]
I've added a rule in 7127cb9. I'd like to keep all issues open for now, in case we want to have some further discussion later on.
